app/controllers/LastUpdateController.scala

```diff
@@ -22,8 +19,8 @@ class LastUpdateController @Inject() (implicit val config: Config) extends Contr
   @ApiResponses(Array(
     new ApiResponse(code = 200, message = "Success - Displays json list of dates for official development.")
   ))
-  def latestListings: Action[AnyContent] = Action.async {
-    Ok(Json.obj("status" -> "OK", "bi-api-deployed-date" -> s"${BuildInfo.builtAtMillis}")).future
+  def latestListings: Action[AnyContent] = Action {
+    Ok(Json.obj("status" -> "OK", "bi-api-deployed-date" -> s"${BuildInfo.builtAtMillis}"))
```

Comment:
Refers to bi-api? In any case as far as I can see this controller is never used so shall we just delete it?